integrations/data.js

```diff
@@ -1 +1 @@
toria-metrics-k8s-stack-white.svg","permalink":"https://docs.meshery.io/extensibility/integrations/victoria-metrics-k8s-stack"},{"name":"vineyard","color":"meshery.io/assets/images/integration/vineyard/icons/color/vineyard-color.svg","white":"meshery.io/assets/images/integration/vineyard/icons/white/vineyard-white.svg","permalink":"https://docs.meshery.io/extensibility/integrations/vineyard"},{"name":"virtual kubelet","color":"meshery.io/assets/images/integration/virtual-kubelet/icons/color/virtual-kubelet-color.svg","white":"meshery.io/assets/images/integration/virtual-kubelet/icons/white/virtual-kubelet-white.svg","permalink":"https://docs.meshery.io/extensibility/integrations/virtual kubelet"},{"name":"vitess","color":"meshery.io/assets/images/integration/vitess/icons/color/vitess-color.svg","white":"meshery.io/assets/images/integration/vitess/icons/white/vitess-white.svg","permalink":"https://docs.meshery.io/extensibility/integrations/vitess"},{"name":"volcano","color":"meshery.io/assets/images/integration/volcano/icons/color/volcano-color.svg","white":"meshery.io/assets/images/integration/volcano/icons/white/volcano-white.svg","permalink":"https://docs.meshery.io/extensibility/integrations/volcano"},{"name":"wasmcloud","color":"meshery.io/assets/images/integration/wasmcloud/icons/color/wasmcloud-color.svg","white":"meshery.io/assets/images/integration/wasmcloud/icons/white/wasmcloud-white.svg","permalink":"https://docs.meshery.io/extensibility/integrations/wasmcloud"},{"name":"wasmedgeruntime","color":"meshery.io/assets/images/integration/wasmedgeruntime/icons/color/wasmedgeruntime-color.svg","white":"meshery.io/assets/images/integration/wasmedgeruntime/icons/white/wasmedgeruntime-white.svg","permalink":"https://docs.meshery.io/extensibility/integrations/wasmedgeruntime"},{"name":"whereabouts","color":"meshery.io/assets/images/integration/whereabouts/icons/color/whereabouts-color.svg","white":"meshery.io/assets/images/integration/whereabouts/icons/white/whereabouts-white.svg","permalink":"https://docs.meshery.io/extensibility/integrations/whereabouts"},{"name":"windows","color":"meshery.io/assets/images/integration/windows/icons/color/windows-color.svg","white":"meshery.io/assets/images/integration/windows/icons/white/windows-white.svg","permalink":"https://docs.meshery.io/installation//windows"},{"name":"wordpress-operator","color":"meshery.io/assets/images/integration/wordpress-operator/icons/color/wordpress-operator-color.svg","white":"meshery.io/assets/images/integration/wordpress-operator/icons/white/wordpress-operator-white.svg","permalink":"https://docs.meshery.io/extensibility/integrations/wordpress-operator"},{"name":"azure-architecture","color":"meshery.io/assets/images/integration/azure-architecture/icons/color/azure-architecture-color.svg","white":"meshery.io/assets/images/integration/azure-architecture/icons/white/azure-architecture-white.svg","permalink":"https://docs.meshery.io/extensibility/integrations/azure-architecture"}];
```

Comment:
@saurabhraghuvanshii, this is likely to break the very next day. The source of truth for this data and it's structure is not defined here.